amr-wind/wind_energy/ABLBoundaryPlane.H

```diff
@@ -7,6 +7,10 @@
 #include "amr-wind/utilities/ncutils/nc_interface.H"
 #include <AMReX_BndryRegister.H>
 
+#ifdef ERF_AW_MULTIBLOCK
+#include "amr-wind/wind_energy/ABLReadERFFcn.H"
+class MultiBlockContainer;
```

Comment:
are these needed? Like the build breaks without them? I see you do it in a bunch of places.

Reply:
do you mean the ifdefs? I could remove them from all places where they are not strictly required. My thinking was that maybe you would prefer using the ifdefs to keep out the multiblock stuff as much as possible.

Reply:
Sorry, I mean the forward declarations.